Source/SIMPLib/Messages/AbstractErrorMessage.h

```diff
@@ -0,0 +1,84 @@
+/* ============================================================================
+ * Copyright (c) 2009-2016 BlueQuartz Software, LLC
```

Comment:
2019-2019

Reply:
2009-1019 since the file was first created back in 2009

Reply:
Can we go back to the future real quick.  This isn't the direction I wanted my life to go.